src/action/common/configure_init_service.rs

```diff
@@ -539,19 +531,14 @@ impl Action for ConfigureInitService {
 
         match self.init {
             InitSystem::Launchd => {
-                execute_command(
-                    Command::new("launchctl")
-                        .process_group(0)
-                        .arg("bootout")
-                        .arg(
-                            [
-                                DARWIN_LAUNCHD_DOMAIN,
-                                self.service_name
-                                    .as_ref()
-                                    .expect("service_name should be defined for launchd"),
-                            ]
-                            .join("/"),
-                        ),
+                crate::action::macos::retry_bootout(
+                    DARWIN_LAUNCHD_DOMAIN,
+                    self.service_name
```

Comment:
the `retry_bootout` function is passing `domain` and **`service_path`** as separate arguments, despite the original here using `domain` and **`service_name`**` joined by a `/`.

Going to pay attention to this for the rest of this review...

Reply:
Yes this is a change, but it should be fine.